packages/serverpod/lib/src/database/analyze.dart

```diff
@@ -141,17 +141,21 @@ WHERE t.relname = '$tableName' AND n.nspname = '$schemaName';
 ''');
 
     return queryResult.map((index) {
+      var indkeyNames = index[4];
```

Comment:
Nit: Is `indKeyNames` suppose to mean `indexKeyNames`? If so I think writing out the entire name is better as it becomes more clear!

Reply:
It is camelification of the actual column name in the SQL query, which I think is good practice in order to be clear which result column is mapped to which variable.